digitalfabrik/entitlementcard
# 1216: Group applications by status

### Short description

As a service user I want a simplified view for each application status to identify faster applications to process or delete

### Proposed changes



- add application filter bar
- group applications by status
- add status withdraw

### Side effects



- none

### Resolved issues



Fixes: #1216 

### Notes

- slightly adjusted colors to fit to the general look
- added some count to the each group
- changed the bar item order according to sorting (check comment in the design)

<img width="933" alt="Bildschirmfoto 2024-05-21 um 15 24 01" src="https://github.com/digitalfabrik/entitlementcard/assets/37902063/e011faba-b117-4a5e-93ca-5afc0500bde8">

### Testing
- Go to "Eingehende Anträge"
- click on the several filter buttons and check if the list items are correct in this group
- check the count of applications of each group
- check if you can properly set/update notes
- check if new applications will be added to the correct group
- check if deleting applications still work correctly
- check the info icon shows explanation to each status
